apps/condo/domains/common/components/containers/BaseLayout/components/SideMenu.tsx

```diff
@@ -97,6 +99,12 @@ const TicketCreateButton = () => {
 
 export const SideMenu: React.FC<ISideMenuProps> = (props) => {
     const { onLogoClick, menuData, isMobile, isSideMenuCollapsed, toggleSideMenuCollapsed } = props
+    const organization = useOrganization()
+    const isEmployeeBlocked = get(organization, ['link', 'isBlocked'], false)
+
+    if (isEmployeeBlocked) {
```

Comment:
Why do you want to hide the side menu for blocked users? probably it's better to patch OrganizationRequired to check it and redirect the user to some page.

Reply:
This behavior is confusing the user.